R/cols-best_case_worst_case.R

```diff
@@ -0,0 +1,43 @@
+col_companies_id <- function() {
+  "companies_id"
+}
+
+col_ep_product <- function() {
+  "ep_product"
+}
+
+col_benchmark <- function() {
+  "benchmark"
+}
```

Comment:
The most general concept behind this column is that of "grouping". In tiltIndicator and tiltAddCO2 we call it `grouped_by`.